packages/geo/src/lib/geometry/geometry-form-field/geometry-form-field-input.component.ts

```diff
@@ -239,7 +261,7 @@ export class GeometryFormFieldInputComponent implements OnInit, OnDestroy, Contr
    */
   private toggleControl() {
     this.deactivateControl();
-    if (!this.value) {
+    if (!this.value && this.geometryType) {
```

Comment:
I get why you check if the geometryType is defined but I think it would be more "futureproof" to check if the drawControl itself is defined.

Reply:
Draw control was still defined even with invalid geometry type

Reply:
I think this would be solved with the modification suggested above.